packages/web-components/src/button/button.options.ts

```diff
@@ -10,7 +10,6 @@ export const ButtonAppearance = {
   primary: 'primary',
   outline: 'outline',
   subtle: 'subtle',
-  secondary: 'secondary',
```

Comment:
is `secondary` just `default` now?

Reply:
I think this was residual from v2 or copy/pasta - I'd probably say that secondary is "default" but we don't "name" it in Fluent. That IMO could present problems if we want to make `default` => `primary`...but it's a larger Fluent problem. Just keeping aligned here.